src/MLJSerialization.jl

```diff
@@ -1,9 +1,19 @@
 module MLJSerialization
 
+using MLJModelInterface
+using MLJBase
+using Serialization
+
+import MLJBase: machine
+import MLJModelInterface: save, restore
+import IterationControl
+
+export serializable, restore!, save, machine
+
```

Comment:
Currently `save` is not exported. Are you wanted to export it for convenience?

Reply:
Yes I was actually wondering why it wasn't?